app/mailers/patient_mailer.rb

```diff
@@ -187,16 +187,50 @@ def assessment_email(patient)
   end
 
   def closed_email(patient)
-    return if patient&.email.blank?
+    if patient&.email.blank?
+      History.send_close_conact_method_blank(patient: patient, type: 'email')
+      return
+    end
 
-    @patient = patient
     @lang = patient.select_language
+    @contents = I18n.t(
+      'assessments.sms.closed.thank-you',
+      initials_age: patient&.initials_age('-'),
+      completed_date: patient.closed_at&.strftime('%m-%d-%Y'),
+      locale: @lang
+    )
     mail(to: patient.email&.strip, subject: I18n.t('assessments.email.closed.subject', locale: @lang)) do |format|
       format.html { render layout: 'main_mailer' }
     end
     History.monitoring_complete_message_sent(patient: patient)
   end
 
+  def closed_sms(patient)
+    if patient&.primary_telephone.blank?
+      History.send_close_conact_method_blank(patient: patient, type: 'primary phone number')
+      return
+    end
+    if patient.blocked_sms
+      TwilioSender.handle_twilio_error_codes(patient, TwilioSender::TWILIO_ERROR_CODES[:blocked_number][:code])
```

Comment:
`handle_twilio_error_codes` will result in history items being added to active dependents in addition  to  the responder when `patient.preferred_contact_method != 'SMS Texted Weblink` this differs from how `send_close_conact_method_blank` is currently operating which only adds a failed history item to  the responder. If we are not going to be sending closed_sms for dependents, then I think now might be a good time to  add that history item referenced here https://github.com/SaraAlert/SaraAlert/pull/815/files#diff-3377084457c29d600d0124cde3cd20f7233e38c40529fe0d9360e77dfdb36875R58 so that the failed history items for closed_sms are distributed consistently for blank contact type and blocked sms.

Reply:
Agreed

Reply:
@mayerm94 @holmesie @Bialogs Please see the most recent changes to add that history item.